Fixtures/IntegrationTests/Mocks/MUXSDKCore+Mock.h

```diff
@@ -0,0 +1,27 @@
+//
+//  MUXSDKCore+Mock.h
+//  MUXSDKStatsTests
+//
+//  Created by Nidhi Kulkarni on 2/4/20.
+//  Copyright © 2020 Mux, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <MuxCore/MuxCore.h>
+
+@interface MUXSDKCore (Mock)
+
++ (void) swizzleDispatchEvents;
++ (void) resetCapturedEvents;
++ (id<MUXSDKEventTyping>) eventAtIndex:(NSUInteger) index forPlayer:(NSString *)playerId;
++ (NSUInteger) eventsCountForPlayer:(NSString *)playerId;
++ (MUXSDKDataEvent *) globalEventAtIndex:(NSUInteger)index;
++ (NSUInteger) globalEventsCount;
++ (NSArray<MUXSDKDataEvent *> *) snapshotOfGlobalEvents;
++ (NSArray<MUXSDKDataEvent *> *) snapshotOfEventsForPlayer:(NSString *)playerId;
++ (NSArray<MUXSDKBaseEvent *> *) getEventsForPlayer: (NSString *)playerId;
++ (NSArray *)getTimeStampsForPlayer:(NSString *)playerId;
```

Comment:
If these are counting playhead deltas, I think we need to name them this way. Like `getPlayheadTimestampsForPlayer` or something.

We also count 'real'/'wall-clock' time for some of our metrics (eg, rebuffering time or seek latency). We are probably going to want to test our actual metric values at some point, so this code may evenutally need to refer to both playhead and wall-clock timestamps unambiguously.